serverless.yml

```diff
@@ -1,125 +1,78 @@
-        TableName: ${self:provider.environment.OA_DOC_STORAGE_TABLE}
-        TimeToLiveSpecification:
-          AttributeName: ttl
-          Enabled: true
-
+        BucketName: '${self:custom.BUCKET_NAME.${self:provider.stage}}'
 functions:
-  email:
-    description: Sends email on behalf of user to third party with the OpenCerts' preset template.
-    handler: src/email.handler
-    events:
-      - http:
-          path: /email
-          method: post
-          cors: true
-  verify:
-    description: Checks the validity of the certificate against the 4 checks.
-    handler: src/verify.handler
-    events:
-      - http:
-          path: /verify
-          method: post
-          cors: true
-  get: 
-    handler: src/storage/get.handler
-    events:
-      - http:
-          path: /storage/get/{id}
-          method: get
-          request:
-            parameters:
-              paths:
-                id: true
-          cors: true
-  create:
-    handler: src/storage/create.handler
-    events:
-      - http:
-          path: /storage/create
-          method: post
-          cors: true
-  getQueueNumber:
-    handler: src/storage/queueNumber.handler
-    events:
-      - http:
-          path: /storage/queue
-          method: get
-          cors: true
-
+    email:
+        description: 'Sends email on behalf of user to third party with the OpenCerts'' preset template.'
+        handler: src/email.handler
+        events:
+            - {http: {path: /email, method: post, cors: true}}
+    verify:
+        description: 'Checks the validity of the certificate against the 4 checks.'
+        handler: src/verify.handler
+        events:
+            - {http: {path: /verify, method: post, cors: true}}
+    get:
+        handler: src/storage/get.handler
+        events:
+            - {http: {path: '/storage/get/{id}', method: get, request: {parameters: {paths: {id: true}}}, cors: true}}
+    create:
+        handler: src/storage/create.handler
+        events:
+            - {http: {path: /storage/create, method: post, cors: true}}
+    getQueueNumber:
+        handler: src/storage/queueNumber.handler
+        events:
+            - {http: {path: /storage/queue, method: get, cors: true}}
 custom:
-  # Enable auto migration in local dynamodb on start
-  dynamodb:
-    start:
-      port: 8000
-      inMemory: true
-      migrate: true
-    migration:
-      dir: offline/migrations
-  SES_KEY_ID:
-    stg: ${env:stg_SES_KEY_ID}
-    prd: ${env:prd_SES_KEY_ID}
-  SES_SECRET:
-    stg: ${env:stg_SES_SECRET}
-    prd: ${env:prd_SES_SECRET}
-  SES_REGION:
-    stg: ${env:stg_SES_REGION}
-    prd: ${env:prd_SES_REGION}
-  RECAPTCHA_SECRET:
-    stg: ${env:stg_RECAPTCHA_SECRET}
-    prd: ${env:prd_RECAPTCHA_SECRET}
-  NETWORK:
-    stg: ${env:stg_NETWORK}
-    prd: ${env:prd_NETWORK}
-  EMAIL_API_KEYS:
-    stg: ${env:stg_EMAIL_API_KEYS}
-    prd: ${env:prd_EMAIL_API_KEYS}
-
-
+    bucket: '${self:custom.BUCKET_NAME.${self:provider.stage}}'
```

Comment:
we need to turn off versioning so that old copies of objects aren't kept
https://forum.serverless.com/t/how-to-specify-s3-bucket-versioningconfiguration-in-serverless-yml/7103

Reply:
Buckets can be in one of three states: unversioned (the default), versioning-enabled, or versioning-suspended

Reply:
by default its off.